### PR TITLE
Some tutorials on the use of Fink

### PR DESCRIPTION
**Pull request by JulienPeloton**
Link to issue: #1 

This PR proposes to add some basic tutorials on the use of Fink.

### CODE REVIEW

File: docs/tutorials/raw_db.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+```
+
+Now go to the dashboard live tab, and see the alerts coming! The tab should automatically refreshed every 2 seconds, but in case hit the refresh button.
+
+## Retrieving archived alert data
+
+The `stream2raw` service connects to the stream, decode the alert data (Apache Avro file format), and archive on disk (Apache Parquet file format). You can easily access it in a pyspark shell:
+
+```bash
+# Launch a pyspark shell with fink dependencies loaded
```
[the remark this answers is not in the document]
Comment (JulienPeloton) [reply docs/tutorials/raw_db.md]: Good point @cAbhi15, fixed in https://github.com/astrolabsoftware/fink/pull/3/commits/dfb02822ede0e323fc30ddce02ba48cbdf67aed1

File: docs/tutorials/raw_db.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+# |      3.2|ZTF (www.ztf.calt...|ZTF18abupgps|697252383615015001|[2458451.7523843,...|[[2458423.7417014...|[candid6972523836...|[candid6972523836...|[candid6972523836...|
+# |      3.2|ZTF (www.ztf.calt...|ZTF18acsbjlx|697251921515015007|[2458451.7519213,...|[[2458422.8142593...|[candid6972519215...|[candid6972519215...|[candid6972519215...|
+# |      3.2|ZTF (www.ztf.calt...|ZTF18abomhfb|697252845215010000|[2458451.7528472,...|[[2458422.7657755...|[candid6972528452...|[candid6972528452...|[candid6972528452...|
+# |      3.2|ZTF (www.ztf.calt...|ZTF18acsbtjt|697252381715015002|[2458451.7523843,...|[[2458423.7417014...|[candid6972523817...|[candid6972523817...|[candid6972523817...|
+# |      3.2|ZTF (www.ztf.calt...|ZTF18abomlbr|697252384915015004|[2458451.7523843,...|[[2458423.7417014...|[candid6972523849...|[candid6972523849...|[candid6972523849...|
+# +---------+--------------------+------------+------------------+--------------------+--------------------+--------------------+--------------------+--------------------+
+```
+
+You can keep accessing the fields but going deeper in the nested structure:
```
[the remark this answers is not in the document]
Comment (JulienPeloton) [reply docs/tutorials/raw_db.md]: Thanks, fixed.

File: docs/tutorials/redistributing_alerts.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+# in conf/fink.conf.tutorial4
+DISTRIBUTION_RULES_XML=${FINK_HOME}/conf/distribution-rules_rrlyr.xml
+```
+
+Then create the Kafka topic "fink_RRLyr":
+
+```bash
+# Create a topic
+fink_kafka --create-topic fink_RRLyr
+```
```
[the remark this answers is not in the document]
Comment (JulienPeloton) [reply docs/tutorials/redistributing_alerts.md]: Thank you for pointing out it! I will merge this PR, and open an issue to not forget to update the tutorial once the PR is merged.

**Comment by JulienPeloton** [comment]
Thanks @cAbhi15 for the review! If you think it is ok, I will merge the PR.

**Comment by cAbhi15** [comment]
Sure! @JulienPeloton go ahead.

**Comment by JulienPeloton** [comment]
Thanks @cAbhi15, merging!

**Pull request merged**